src/control/mod.rs

```diff
@@ -834,18 +843,17 @@ bitflags::bitflags! {
     }
 }
 
-#[repr(u32)]
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 /// Target to alter the sequence of page flips
 ///
-/// These are the [`ffi::drm_sys::DRM_MODE_PAGE_FLIP_FLAGS`] bits
+/// These represent the [`ffi::drm_sys::DRM_MODE_PAGE_FLIP_TARGET`] bits
```

Comment:
Nice catch ^^ (not sure why it was written correctly in the docs for `PageFlipFlags` above) :+1: